snuba/querylog/__init__.py

```diff
@@ -38,8 +38,15 @@ def record_query(
             mark_tags={"final": final},
         )
 
-        if Hub.current.scope.span:
-            sentry_sdk.set_tag("duration_group", timer.get_duration_group())
+        _add_tags(timer)
+
+
+def _add_tags(timer: Timer) -> None:
+    if Hub.current.scope.span:
+        duration_group = timer.get_duration_group()
+        sentry_sdk.set_tag("duration_group", duration_group)
+        if duration_group != "<10s":
+            sentry_sdk.set_tag("timeout", "true")
```

Comment:
10+ and 20+ seconds should not be considered timeout. The user is getting an answer.
If you are actually trying to track queries that are too slow, I would call this tag in a different way.

Reply:
I thought we talked about tracking >10s and >20s as well for query performance purposes? I'm trying to add a tag that allows us to see the impact of query performance changes. Would it help if I renamed this tag?

Reply:
OK I changed this tag to try and capture the cases you listed above. I didn't capture "No value at key" errors since those are not necessarily timeout errors.